### PR TITLE
Start a bibliography

### PR DESCRIPTION
**Pull request by pjz**
Here's links to some of the prominent papers.  I put them in a separate file, but maybe they should be incorporated into the .pdfs?  Or copies of the papers should be downloaded and put in a library section?  Your call.

### CODE REVIEW

File: bibliography.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+Toward a readable IPFS Bibliography:
+====================================
+
+   * [Kademlia: A Peer-to-peer Information System Based on the XOR Metric](http://pdos.csail.mit.edu/~petar/papers/maymounkov-kademlia-lncs.pdf) by Petar Maymounkov and David Mazieres
+   * [S/Kademlia: A Practicable Approach Towards Secure Key-Based Routing](http://www.researchgate.net/publication/4319659_SKademlia_A_practicable_approach_towards_secure_key-based_routing/links/02e7e524ad3e97d67d000000) by Ingmar Baumgart and Sebastian Mies
+   * [Escaping the Evils of Centralized Control with self-certifying pathnames](http://www.scs.stanford.edu/~dm/home/papers/mazieres:escape.ps.gz)
```
Comment (RichardLitt) [inline bibliography.md]: Maybe link http://www.sigops.org/ew-history/1998/papers/mazieres.ps instead, and add in `by David Mazieres`

Comment (RichardLitt) [inline bibliography.md]: @pjz Any word on this?

**Comment by pjz** [comment]
There ya go; sorry, I thought this had been merged long ago.

**Comment by RichardLitt** [comment]
@pjz Sorry for the delay, again! 😓 

I've been thinking about this. I don't want ipfs/ipfs to turn into a reading list, at the moment, because that will end up with a lot of edits that aren't directly related to IPFS, and I want this repository to be fairly clean. Part of that process is going through and closing old issues, something I have in my pipeline to work on. 

I've started the [ipfs/reading-list](https://github.com/ipfs/reading-list) and plan to merge this into that repository. I can do this manually and attribute you in the commit, if you like, or you can make a PR there and this can be closed. Which would you prefer? 

Does this make sense, to you? Hopefully, we can have more papers and engagement there, where the reading list won't be drowned amidst tons of links.

**Comment by pjz** [comment]
Do whatever you think is best. I'm not so attached to the idea that I want to bother defending it.

**Comment by RichardLitt** [comment]
Closing in favor of ipfs/reading-list#4. Thank you, @pjz.

**Pull request closed without merging**